config/clusters/2i2c/binder-staging.values.yaml

```diff
@@ -1,11 +1,34 @@
 binderhub:
   config:
     BinderHub:
+      auth_enabled: true
       template_path: /etc/binderhub/custom/templates
       extra_static_path: /etc/binderhub/custom/static
       extra_static_url_prefix: /extra_static/
       about_message: |
         <p>binder.pangeo.io is public infrastructure operated by the <a href="https://2i2c.org">2i2c team</a>.<br /><br />
+  jupyterhub:
+    hub:
+      redirectToServer: false
+      config:
+        BinderSpawner:
+          auth_enabled: false
+        services:
+          binder:
+            oauth_no_confirm: true
+            oauth_redirect_uri: "https://binder-staging.2i2c.cloud/hub/oauth_callback"
+        JupyterHub:
+          authenticator_class: cilogon
+        CILogonOAuthenticator:
+          oauth_callback_url: "https://binder-staging.2i2c.cloud/hub/oauth_callback"
+          username_claim: "email"
+          allowed_idps:
+            - http://google.com/accounts/o8/id:
+                username_derivation:
+                  username_claim: "email"
+          username_pattern: '^(.+@2i2c\.org|deployment-service-check)$'
```

Comment:
@sgibson91, maybe adding the `username_pattern` under `hub.config.Authenticator` like in

https://github.com/2i2c-org/infrastructure/blob/22c4423484e08e6235128e63339f3606ddb8595a/config/clusters/2i2c/dask-staging.values.yaml#L50-L55 ?

When I made the suggestion of adding it here, I assumed it would work this way too, since CILogonOAuthenticator is inheriting from the base Authenticator

Reply:
Thank you, will try that

I also seem to have acquired two instances of username_claim like this:

```
        CILogonOAuthenticator:
          oauth_callback_url: "https://binder-staging.2i2c.cloud/hub/oauth_callback"
          username_claim: "email"
          allowed_idps:
            - http://google.com/accounts/o8/id:
                username_derivation:
                  username_claim: "email"
```

I'm assuming one of these is wrong?